test/src/unit-filter-pipeline.cc

```diff
@@ -5,7 +5,7 @@
  *
  * The MIT License
  *
- * @copyright Copyright (c) 2017-2018 TileDB, Inc.
+ * @copyright Copyright (c) 2017-2018 tiledb::sm::TileDB, Inc.
```

Comment:
This file can be reverted (looks like find/replace errors).

Reply:
This is wrong for sure, but I am not sure if the entire file can be reverted. A lot of places I had to add proper namespace for objects because now two namespaces exist `tiledb::sm` and `tiledb::rest`.

Reply:
I'll revert it for now and add it back later if it's a problem.